CMSgov/mdct-mcr
# MDCT-2041: Page-level status indicators

## Description
Fixes [MDCT-2041](https://qmacbis.atlassian.net/browse/MDCT-2041)

Adds page level status indicators if you've completed a plan on a DrawerReport page!

https://user-images.githubusercontent.com/19439679/199075358-1e6f1568-7207-4d07-a456-ebc1e89ad6fc.mov

### How to test

1. Create a program
2. Go to Section A: Add Plans
3. Create a plan or as many plans as you like
4. Navigate to Section D: Program Characteristics
5. Complete one of the plans you've created by hitting enter on the plan and filling out the drawer
6. Note that the plan you entered info for and saved and closed now has a green checkmark and any other plans you created dont
7. Nagivate to Section D: Financial Performance (Or any other section in D other then Quality Measures and Sanctions)
8. Repeat 5-6 as much as you like!

### Changed Dependencies


## Code author checklist

## Reviewer checklist (two different people)
